### PR TITLE
improved resolution of Now and UtcNow

### LINKED ISSUE #2623: Resolution of DateTime.Now and UtcNow
I think the documentation of resolution of `DateTime.Now` and `DateTime.UtcNow` could use some improvement.

Currently, [the docs for `DateTime.Now` say](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.now?view=netframework-4.7#remarks):

> The resolution of this property depends on the system timer, which is approximately 15 milliseconds on Windows systems.

[Docs for `DateTime.UtcNow`](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.utcnow?view=netframework-4.7#remarks):

>  The resolution of this property depends on the system timer.  
>
> |System|Approximate Resolution|  
> |------------|----------------------------|  
> |Windows NT 3.5 and later|10 milliseconds|  
> |Windows 98|55 milliseconds|  

My thoughts:

* What Windows 98 or Windows NT 3.5 did does not matter anymore and should not be in the current version of docs.
* Do `Now` and `UtcNow` really have different resolution (15 ms vs. 10 ms)?
* The resolution of `Now` and `UtcNow` seems to be 0.5 ms on my Windows 10 machine.
* Documentation for Linux and macOS would be useful too.

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #2623 

/cc @svick

### CODE REVIEW

**Review by tarekgh — APPROVED** [review]

File: xml/System/DateTime.xml [hunk 1]
```diff
@@ -7008,18 +7008,11 @@ An object that is equivalent to the date and time contained in `s`.
...
-|Windows 98|55 milliseconds|  
-  
- Starting with the .NET Framework version 2.0, the return value is a <xref:System.DateTime> whose <xref:System.DateTime.Kind%2A> property returns <xref:System.DateTimeKind?displayProperty=fullName>.  
-  
- An alternative to using <xref:System.DateTime.UtcNow%2A> is <xref:System.DateTimeOffset.UtcNow%2A?displayProperty=fullName>. While the former indicates that a date and time value is Coordinated Universal Time (UTC) by assigning <xref:System.DateTimeKind?displayProperty=fullName> to its <xref:System.DateTime.Kind%2A> property, the latter assigns the date and time value the UTC time's offset (equal to <xref:System.TimeSpan.Zero?displayProperty=fullName>).  
-  
-   
+ The resolution of this property depends on the system timer, which depends on the underlying operating system. It tends to be between 0.5 and 15 milliseconds.
+
+ Starting with the .NET Framework version 2.0, the return value is a <xref:System.DateTime> whose <xref:System.DateTime.Kind%2A> property returns <xref:System.DateTimeKind?displayProperty=fullName>.
```
Comment (svick) [inline xml/System/DateTime.xml]: > the return value is a DateTime whose Kind property returns System.DateTimeKind. 

Unrelated to this change, but this looks like another migration issue similar to https://github.com/dotnet/docs/issues/1932: it should specify *which* `DateTimeKind` it returns, and it does [on MSDN](https://msdn.microsoft.com/en-us/library/system.datetime.utcnow(v=vs.110).aspx):

>  the return value is a DateTime whose Kind property returns DateTimeKind.Utc.

Should I open a new issue for this?

**Review by svick — APPROVED** [review]

File: xml/System/DateTime.xml (hunk 1 above)
Comment (mairaw) [reply xml/System/DateTime.xml]: I'll fix this one but please do and I'll follow up with the team. We keep finding new occurrences here and there.

Comment (svick) [reply xml/System/DateTime.xml]: Ok, done: https://github.com/dotnet/docs/issues/3118.

**Review by rpetrusha — APPROVED** [review]
@mairaw, LGTM. I think that this is ready to merge.

**Pull request merged**